src/strategies/dps-nft-strategy/README.md

```diff
@@ -0,0 +1,13 @@
+# erc20-balance-of
+
+This is the most common strategy, it returns the balances of the voters for a specific ERC20 token.
+
+Here is an example of parameters:
+
```

Comment:
Can you add more relevant information here plz :) This doc is for erc20-balance-of

Reply:
Ok, sorry for that! :)